polkadot/node/subsystem-bench/examples/estimate_cpu_requirements.yaml

```diff
@@ -0,0 +1,47 @@
+## Estimate CPU requirements on a network with 500 validators 100 cores
```

Comment:
I think we should merge this file to not have to recreate it every time we want to do such tests.

Reply:
Yeah, that was my plan, I just wanted to see the results first.